toolbox/util/commonUtils.go

```diff
@@ -40,6 +40,27 @@ var (
 	}
 )
 
+// IsLongerThan compares two duration
+func IsLongerThan(durA time.Duration, durB time.Duration) bool {
```

Comment:
Do you really need a function for this ?

Reply:
It is just that calling Nanoseconds() repetitively is not very clean. I removed this function tho.